source/Hazard.cpp

```diff
@@ -34,6 +34,8 @@ void Hazard::Load(const DataNode &node)
 			LoadWeapon(child);
 		else if(key == "constant strength")
 			deviates = false;
+		else if(key == "system-wide")
```

Comment:
Is this keyword needed, or should it be the default when no range was given?

Reply:
The issue I have with that is then we need to choose a default hardcoded range for how far hazards appear from the flagship, whereas having a separate keyword still lets us use the range to adjust how effects appear from within the data files.